src/main/java/com/yurjinia/auth/service/AuthService.java

```diff
@@ -134,6 +142,27 @@ private RegistrationRequest payloadOAuthUser(String email, String firstname, Str
                 .build();
     }
 
+    public void logout(HttpServletRequest request, HttpServletResponse response) {
+        String requestHeader = request.getHeader(JwtConstants.AUTHORIZATION_HEADER);
```

Comment:
I think the implementation can be done as part of Security Configuration. If you check the structure where you can find when jwt authentication filter is configured - there is possibility of adding 'Logout Handler' method.

Ask it through ChatGPT and see if it's possible, otherwise leave as is.